advancedblacklist/const.py

```diff
@@ -0,0 +1,26 @@
+"""Different constants for this cog"""
+
+from typing import Any, Dict
+
+_config_structure: Dict[str, Dict[str, Any]] = {
+    "global": {
+        "blacklist": {},
+        "whitelist": {},
+    },
+    "guild": {
+        "blacklist": {},
+        "whitelist": {},
+    },
+}
+
+__version__ = "2.0.0"
+
+
+class _Lazy(list):
+    """Jojo is lazy"""
+
+    def tick_self(self):
+        return [f"`{x}`" for x in self]
+
+
+__authors__ = _Lazy(["Jojo#7791"])
```

Comment:
Creative, but I don't think reviewers will enjoy this one. You should use something like this instead:

```py
from redbot.core.utils.chat_formatting import inline

__authors__ = list(map(inline, ["Jojo#7791"]))
```
I suggest this change seeing as its never used without calling `tick_self()`.